RELEASE-NOTES.md

```diff
@@ -1,5 +1,65 @@
 # Prefect Release Notes
 
+## Release 2.8.1
+
+### New names, same behavior
+We knew that we were on to something big when we [first announced Prefect Orion](https://www.prefect.io/guide/blog/announcing-prefect-orion/), our second generation orchestration engine, but we didn't know just how big. Orion's foundational design principles of "dynamism, developer experience, and observability" have influenced the entire Prefect 2 codebase to such an extent that it's difficult to tell where Orion ends and other components begin. In particular, it can be difficult to distinguish between the “Orion API” - the orchestration API, “Orion Server” - a hosted instance of that API, and components of that server. 
```

Comment:
```suggestion
We knew that we were on to something big when we [first announced Prefect Orion](https://www.prefect.io/guide/blog/announcing-prefect-orion/), our second-generation orchestration engine, but we didn't know just how big. Orion's foundational design principles of "dynamism, developer experience, and observability" have influenced the entire Prefect 2 codebase to such an extent that it's difficult to tell where Orion ends and other components begin. In particular, it can be difficult to distinguish between the “Orion API” (the orchestration API), “Orion Server” (a hosted instance of that API), and individual components of that server. 
```